packages/bigtable/src/cluster.js

```diff
@@ -0,0 +1,247 @@
+/*!
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*!
+ * @module bigtable/instance
+ */
+
+'use strict';
+
+var common = require('@google-cloud/common');
+var format = require('string-format-obj');
+var is = require('is');
+var nodeutil = require('util');
```

Comment:
Sorry that was a little vague. I meant we name it `util` now, since we access *our* `util` through `common`.